feature-libs/checkout/b2b/core/facade/checkout-payment-type.service.spec.ts

```diff
@@ -118,12 +123,28 @@ describe(`CheckoutPaymentTypeService`, () => {
           done();
         });
     });
+
+    it(`should return an empty array if query's data is falsy`, (done) => {
+      spyOn(service, 'getPaymentTypesState').and.returnValue(
```

Comment:
an example of 2. from DM

left spyOn the actual service, which is not one of the mocked dependency that gets injected to this service.